code/app/src/main/java/com/kernelcrew/moodapp/ui/FollowersFragment.java

```diff
@@ -0,0 +1,135 @@
+package com.kernelcrew.moodapp.ui;
+
+import android.os.Bundle;
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.fragment.app.Fragment;
+import androidx.navigation.Navigation;
+import androidx.recyclerview.widget.LinearLayoutManager;
+import androidx.recyclerview.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.CheckBox;
+import android.widget.TextView;
+
+import com.google.android.material.appbar.MaterialToolbar;
+import com.google.android.material.bottomnavigation.BottomNavigationView;
+import com.google.firebase.firestore.DocumentSnapshot;
+import com.google.firebase.firestore.FirebaseFirestore;
+import com.kernelcrew.moodapp.R;
+import com.google.android.material.imageview.ShapeableImageView;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FollowersFragment extends Fragment {
+
+    private RecyclerView followersRecyclerView;
+    private FollowersAdapter adapter;
+    private List<User> followersList = new ArrayList<>();
+    private FirebaseFirestore db;
+
+    @Override
+    public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
+        View view = inflater.inflate(R.layout.fragment_followers, container, false);
+
+        followersRecyclerView = view.findViewById(R.id.followersRecyclerView);
+        followersRecyclerView.setLayoutManager(new LinearLayoutManager(requireContext()));
+
+        adapter = new FollowersAdapter(followersList);
+        followersRecyclerView.setAdapter(adapter);
+
+        db = FirebaseFirestore.getInstance();
+        fetchFollowers();
+
+        BottomNavigationView bottomNavigationView = view.findViewById(R.id.bottom_navigation);
+        bottomNavigationView.setSelectedItemId(R.id.page_myProfile);
+        bottomNavigationView.setOnItemSelectedListener(item -> {
+            if (item.getItemId() == R.id.page_myProfile) {
+                Navigation.findNavController(view).navigate(R.id.myProfile);
+                return true;
+            }
+            return false;
```

Comment:
The bottom navbar doesn't work in the followers page. Please use the `BottomNavBarController` instead.